cypress/integration/home.spec.ts

```diff
@@ -1,4 +1,57 @@
+import {
+  getInput,
+  getInputField,
+  getAddButton,
+  getTodo,
+  getTodoRemoveButton,
+  getTodoToggleButton,
+} from '../support/po/home.po';
+
 it('should redirect to home page', () => {
   cy.visit('/');
   cy.location('pathname').should('eq', '/');
 });
+
+it('should create a new task', () => {
```

Comment:
Every test should have a `when`, so, I suggest adding here.
```suggestion
it('should create a new task when form is submitted correctly', () => {
```

Reply:
Resolved on 23bea404057f61889564ac2b33a6d65dd5a25245